packages/flutter/test/material/dropdown_test.dart

```diff
@@ -422,6 +422,67 @@ void main() {
     checkSelectedItemTextGeometry(tester, 'two');
   });
 
+  testWidgets('Dropdown menu scrolls to first item in long lists', (WidgetTester tester) async {
+    final Key buttonKey = UniqueKey();
+    Widget build() => buildFrame(
```

Comment:
We typically only use `=>` for one-liners that return a value.

Why not just put the buildFrame() call in-line?